tmp/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
It's for storing the ssh_config and ssh_key files, which will be generated by ursula-cli only when --provisioner=heat, sitecontroller also has this. Without this folder, ursula-cli will fail on writing ssh key.